File: cases/taint/javascript/return-relay-one-hop-positive/ReturnRelayOneHopPositive.js
function dfb_source() { // DFB-SOURCE: return-one-hop-input
  return "tainted";
}

function relay(value) { // DFB-WITNESS: return-one-hop-relay
  return value;
}

function dfb_sink(value) {} // DFB-SINK: return-one-hop-sink

function run() {
  const result = relay(dfb_source());
  dfb_sink(result);
}

File: cases/taint/rust/return-relay-one-hop-positive/return_relay_one_hop_positive.rs
fn dfb_source() -> i32 { // DFB-SOURCE: return-one-hop-input
    1
}

fn relay(value: i32) -> i32 { // DFB-WITNESS: return-one-hop-relay
    value
}

fn dfb_sink(value: i32) {} // DFB-SINK: return-one-hop-sink

fn run() {
    let result = relay(dfb_source());
    dfb_sink(result);
}
